packages/dapp/src/components/newsroom/NewsroomManagementV2.tsx

```diff
@@ -12,6 +12,11 @@ export interface NewsroomManagementState {
 }
 
 export default class NewsroomManagement extends React.Component<NewsroomManagementProps, NewsroomManagementState> {
+  constructor(props: NewsroomManagementProps) {
```

Comment:
why is this constructor needed?

Reply:
needed to set `this.state = {}` otherwise cannot read `metamaskEnabled` state of undefined.